portfolio/style.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: 'Syncopate', sans-serif;
+}
+
+.container-fluid {
+  padding-left: 0px !important;
+  padding-right: 0px !important;
+}
+
+.nav {
+  background: black;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+}
+
+.nav-pills {
+  background: black;
+}
+
+.nav-link {
+  color: white;
+}
+
+.btn {
+  color: black;
+}
+
+.fa-github {
+  color: #b36b00;
+}
+
+.fa-twitter {
+  color: #0066ff;
+}
+
+.fa-facebook-official {
+  color: #006600;
+}
+
+.btn-circle {
+  width: 50px;
+  height: 50px;
+  margin-top: 10px;
+  padding: 5px 10px;
+  border: 2px solid white;
+  border-radius: 100% !important;
+  font-size: 30px;
+  color: white;
+  background: transparent;
+  -webkit-transition: background 0.3s ease-in-out;
+  -moz-transition: background 0.3s ease-in-out;
+  transition: background 0.3s ease-in-out;
+}
+
+.btn-circle:hover,
+.btn-circle:focus {
+  outline: none;
+  color: white;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+h1 {
+  padding: 35px 0px 15px 0px;
+  margin: 0px 0px 10px 0px;
+  font-size: 8vw;
+  font-family: 'Lobster', sans-serif;
+  width: 50%;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 3%;
+  color: #fff;
+}
+
+h2 {
+  padding: 10px;
+  margin: 0px;
+  font-size: 3vw;
+  font-family: 'Lobster', sans-serif;
+  width: 40%;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 3%;
+  color: #fff;
+}
+
+h3 {
+  padding: 10px;
+  margin: 0px;
+  font-size: 1.75vw;
+  letter-spacing: 1px;
+  font-family: 'Open Sans Condensed', sans-serif;
+  width: 70%;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 3%;
+  color: #fff;
+  line-height: 180%;
+}
+
+.btn {
+  background: rgba(82, 82, 122, 0.4);
+}
+
+.portfolio {
```

Comment:
Should this be `.thumbnail` instead of `.portfolio`. You use the `portfolio` class on the title links not the your portfolio "boxes". At the moment, all this is doing is adding some space around the links and screwing the alignment.